agents/agent.py

```diff
@@ -721,3 +721,130 @@ def maybe_add_to_queue(self, data):
 
         if command_history.command in self.whitelist_commands:
             self.queue.put(command_history)
+
+    def get_cpu_usage(self, interval=60):
```

Comment:
Consider making these functions - `get_cpu_usage`, `get_ram_usage` etc. - into static functions. They don't really depend on agents and `ServerAgent` is already bloated with methods.